engine/admin/logging/awslogs.md

```diff
@@ -63,6 +63,16 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### tag
+
+Specify `tag` as an alternative to the `awslogs-stream` option. `tag` interprets template markup (e.g., `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`). 
```

Comment:
You probably want to warp this whole section in these tags:

`{% raw %}`
and
`{% endraw %}`

Otherwise, those instances of curly braces are going to get stripped out.